lib/sms_autofill.dart

```diff
@@ -434,7 +435,7 @@ class _TextFieldPinAutoFillState extends State<TextFieldPinAutoFill> with CodeAu
     if (_textController.text != code) {
       _textController.value = TextEditingValue(text: code ?? '');
       if (widget.onCodeChanged != null) {
-        widget.onCodeChanged(code ?? '');
+        widget.onCodeChanged!(code ?? '');
```

Comment:
Unsure why are using bangs here. Like the fact it returns code or empty.

Reply:
because that method is optional for the end user, so all good here to me:)

Reply:
Would it not be safer to use ?.call if possible?

Reply:
Happy to merge in the meantime, I can resolve my thing in another PR.

Reply:
There is a if ^^ so no it's not safer, just one line instead of two :) just matter of style